go-controller/pkg/ovn/gress_policy.go

```diff
@@ -357,19 +357,21 @@ func (gp *gressPolicy) buildLocalPodACLs(portGroupName string, aclLogging *ACLLo
 	return acls
 }
 
+func getGressPolicyACLName(ns, policyName string, idx int) string {
+	return joinACLName(ns, policyName, fmt.Sprintf("%v", idx))
+}
+
 // buildACLAllow builds an allow-related ACL for a given match
 func (gp *gressPolicy) buildACLAllow(match, l4Match string, ipBlockCIDR int, aclLogging *ACLLoggingLevels) *nbdb.ACL {
-	var direction string
-	var options map[string]string
+	var aclT aclType
 	if gp.policyType == knet.PolicyTypeIngress {
-		direction = nbdb.ACLDirectionToLport
+		aclT = lportIngress
```

Comment:
I think I had this comment before, should we have a `aclTypeFromPolicyType` method?